src/App.tsx

```diff
@@ -1,11 +1,28 @@
-import List from "./List.tsx";
+import React from 'react';
+
+interface ColoredTextProps {
+  text: string;
+  color: string;
+}
+
+const ColoredText: React.FC<ColoredTextProps> = ({ text, color }) => {
+  return <span style={{ color }}>{text}</span>;
+};
+
+const App: React.FC = () => {
+  const texts = [
```

Comment:
Could you please move the texts array to a separate file? This will help keep our code more modular and easier to manage. Thank you!